test/toAdditive.lean

```diff
@@ -110,7 +110,7 @@ run_cmd do
 --         = (add_units.mk_of_add_eq_zero 0 0 (by simp) : ℕ) :=
 -- by normCast
 
-def foo_mul {I J K : Type} (n : ℕ) {f : I → Type} (L : Type) [∀ i (n : ℕ), Bool → One (f i)]
+def foo_mul {I J K : Type} (n : ℕ) {f : I → Type} (L : Type) [∀ i, One (f i)]
```

Comment:
@EdAyers the local instance is now rejected since the `Nat/Bool` params have not forward dependencies so I removed them. Does the test still make sense?

Reply:
This is used to test whether `firstMultiplicativeArg` works. I'm not that sure but if I recall correctly the first multiplicative arg in `foo_mul` is the `n : ℕ` so I think we are ok.